setup.cfg

```diff
@@ -1,3 +1,15 @@
+[isort]
+skip =
+    _vendor
+    __main__.py
+multi_line_output = 5
+known_third_party =
+    pip._vendor
```

Comment:
All packages from `pip._vendor` should be treated like third-party packages and should be grouped with them.